### PR TITLE
anti c4 gamemode flag, anti drag flag fix

### PR DESCRIPTION
**Pull request by cuberound**
# About the pull request

adds togglable flag that prevents c4s to be planeted on humans when it is on gamemode. fixes bipasses of anti drag

# Explain why it's good for the game

c4ing bodies of other side SUCKS and is hated by every side, if you want it for zombies, go ahead I place it on HvH. draging with roller beds and bodybags is not intended and a bug

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: toggleble flag to prevent c4s on human for gamemodes
fix: anti stripmode flag prevents buckling to roller beds and closing into bags to move dead or crit bodies of enemy faction
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/game/gamemodes/round_modifiers.dm [hunk 1]
```diff
@@ -115,3 +115,15 @@
...
+	modifier_name = "Heavy uscm specialists"
+	modifier_desc = "Replaces all specialist kits with b18 armor kit."
+
+/datum/gamemode_modifier/weaker_explosions
+	modifier_name = "Weaker explosions"
+	modifier_desc = "Reduces damage and stun from explosions."
+
+/datum/gamemode_modifier/no_body_c4
+	modifier_name = "No body c4"
+	modifier_desc = "Prevents c4 explosives from being planted on dead body."
```
Comment (harryob) [inline code/game/gamemodes/round_modifiers.dm:129]: am i crazy or do the top two not have an implementation on this branch

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**